runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -773,6 +773,46 @@ else if (option.equals("collect to inventory") || option.startsWith("collect-not
 					break;
 			}
 		}
+		else if (option.equals("value"))
+		{
+			if (config.shopBuy() != BuyMode.OFF)
+			{
+				switch (config.shopBuy())
+				{
+					case BUY_1:
+						swap("buy 1", option, target, index);
+						break;
+					case BUY_5:
+						swap("buy 5", option, target, index);
+						break;
+					case BUY_10:
+						swap("buy 10", option, target, index);
+						break;
+					case BUY_50:
+						swap("buy 50", option, target, index);
+						break;
+				}
```

Comment:
Instead of creating a switch case for each enum value, the menu options can be defined in the enum and set much more concisely here.
```suggestion
				swap(config.shopBuy().getOption(), option, target, index);
```

Reply:
Added this for other similar menu entry swaps.